understanding/22/visible-controls.html

```diff
@@ -63,6 +63,14 @@ <h3>When Needed</h3>
 		<h3>Embedded Controls</h3>
 		
       <p>Information needed to identify user interface components is usually the component itself; however, there are situations where controls are not visible. Complex components such as video players and carousels often include controls that are visible on hover. The presence of the complex component — for example, the carousel component or the video image — provides the “Information needed to identify the user interface components.” For a carousel, controls displayed on hover should also be displayed when the carousel is activated. For a video player, activating the video image should launch the video (sometimes in a new window), at which point more video player controls, including controls displayed previously on hover, should be visible. Some users may still struggle if video controls are not persistently visible, so there is benefit to providing a mechanism that allows users to have the controls persistently visible.</p>
+      
+      <p>Many scenarios are out of scope because they do not rely on hover or focus, for example:</p>
+      <ul>
+         <li>Options/fields become visible because of a <b>selection</b> elsewhere on the form. For example, address and City fields only appearing once the user has selected a State.</li>
+         <li>An e-reader that allows the user to <b>select</b> a text passage and then pops up contextual toolbar with buttons that allows the user to interact with that text.</li>
+         <li>Controls that become visible upon scrolling, such as a confirmation button that only appears once the user has scrolled through all the content in a Terms and Conditions dialog.</li>
```

Comment:
re-reading this, perhaps swap out `once` with `when` (to avoid making it sound like it's anything to do with "only show this 1 time")